compose.free.yaml

```diff
@@ -14,7 +14,7 @@ x-omnistrate-image-registry-attributes:
 
 services:
   hello-world:
-    image: ghcr.io/omnistrate/ci-cd-example:latest
+    image: ghcr.io/omnistrate/ci-cd-example:latest # this does not work, because at the moment you push to prod, we need to replace the version during release
```

Comment:
Let me address this. We can use a placeholder here like Falkor is doing https://github.com/FalkorDB/falkordb-omnistrate/blob/d5200b948382e4df7e44c4ece05389307d5313e4/omnistrate.free.yaml#L77

Reply:
Addressed